### DISCUSSION TITLE
Dynamic step changes while dragging Slider and RangeSlider

### DISCUSSION THREAD

**Discussion by aaronzomback** [mantinedev/mantine#1055] [Feature requests]
When implementing a conditional in the step prop on Slider, i.e.

`step={investmentValue >= 1000 ? 100 : 10}`

The step changes only using the keyboard arrows, but not with mouse drag on slider.

**Comment by rtivital** [comment]
Thanks for leaving a feature request, I'll be working on it in one of the releases after 3.5, leaving it for community now, so feel free to contribute it if "Help wanted" label is still on this issue
